signingscript/sign.py

```diff
@@ -116,9 +124,13 @@ def build_signtool_cmd(context, from_, fmt, to=None):
         str: the path to the signed file
 
     """
-    log.info("sign_file(): signing {} with {}...".format(from_, fmt))
-    cmd = build_signtool_cmd(context, from_, fmt, to=to)
-    await utils.execute_subprocess(cmd)
+    if utils.is_autograph_signing_format(fmt):
```

Comment:
:+1: I love the fact this happens at this layer \o/ It's gonna be easier to let autograph sign APKs :smiley: